charts/argo-cd/Chart.yaml

```diff
@@ -3,7 +3,7 @@ appVersion: v2.7.0
 kubeVersion: ">=1.22.0-0"
 description: A Helm chart for Argo CD, a declarative, GitOps continuous delivery tool for Kubernetes.
 name: argo-cd
-version: 5.30.1
+version: 5.31.0
```

Comment:
In order to follow upstream (https://github.com/argoproj/argo-cd/commit/c2e0026de16c2622ddf4c95d0311c321ebd51f3a) `.Values.global.entrypoint.useImplicit` is not used anymore.
So I incremented minor version.